src/components/utils/hooks/useStreamMessages.tsx

```diff
@@ -0,0 +1,57 @@
+import { Conversation, Message, Stream } from '@xmtp/xmtp-js';
+import { useEffect, useState } from 'react';
+import { useXmtpStore } from 'src/store/xmtp';
+
+const useStreamMessages = (conversation: Conversation, onMessageCallback: () => void) => {
```

Comment:
@bhavya2611 I might be missing something, but I don't see this being used right now. Was this meant for the `Messages` component where we are streaming currently?

Reply:
Hey we will need this in near future, so having this makes sense